gridstatus/caiso.py

```diff
@@ -1484,10 +1495,17 @@ def _get_historical(
     # but has nulls for all other columns
     df = df.dropna(subset=df.columns[1:], how="all")
 
+    # sometimes this data is from the previous day, in which case we want to label it as such
+    latest_file_time = utils.check_latest_value_time(df, file)
+    current_caiso_time = pd.Timestamp.now(tz=CAISO.default_timezone)
+
+    if latest_file_time > current_caiso_time:
+        date = date - pd.Timedelta(days=1)
```

Comment:
Do we need this logic for all datasets that use `_get_historical`, or only for the two